Review.java

```diff
@@ -239,4 +239,35 @@ public static String fakeReview(String fileName)
       }
       return text;
    }
+  
+  public static String fakeReviewStronger(String fileName)
+  {
+      String text = textToString(fileName);
+      
+      for(int i = 0; i < text.length(); i++)
+      {
+         if(text.charAt(i) == '*')
+         {
+            int j = i;
+            while(true)

```

Comment:
this loop is potentially unsafe if there are no spaces in the string, you will get a string out of bounds error